src/ucs/arch/x86_64/cpu.h

```diff
@@ -112,6 +115,14 @@ static inline void *ucs_memcpy_relaxed(void *dst, const void *src, size_t len,
         return dst;
     }
 #endif
+
+#if ((ENABLE_NT_BUFFER_TRANSFER) && (__AVX__))
```

Comment:
can remove all ( ) here

Reply:
Removing the entire "#if ((ENABLE_NT_BUFFER_TRANSFER) && (__AVX__))" will make the normal compilation to fail. For nt-buffer-transfer to get compiled we need the '-mavx' flag.

I need at least "#if __AVX__" .

Reply:
check only AVX condition